docs/tempo/website/configuration/multitenancy.md

```diff
@@ -48,7 +48,7 @@ exporters:
 - The way the read path is configured is temporary and should be much more straightforward once the [tempo-query dependency is removed](https://github.com/grafana/tempo/issues/382).
 
 ## Enabling Multitenancy
-If this is desired simply set the following config value on all Tempo components:
+To enable multitenancy on Tempo backend, simply set the following config value on all Tempo components:
```

Comment:
```suggestion
To enable multitenancy on Tempo backend, refer to the following config value on all Tempo components:
```

Reply:
I think the previous sentence was good here 👍